codespell_lib/data/dictionary.txt

```diff
@@ -18116,7 +18116,7 @@ mathematicas->mathematics
 mathes->matches
 mathetician->mathematician
 matheticians->mathematicians
-mathimatic->mathematic
+mathimatic->mathematics
```

Comment:
I'm not too sure about this one, because `mathematic` (which you removed) is an alternative form of the adjective `mathematical` (e.g. "a mathematical equation").

Reply:
Thank you, I have reverted back. Found in https://www.lexico.com/en/definition/mathematical  (also mathematic).